airbyte-cdk/python/pyproject.toml

```diff
@@ -46,7 +46,8 @@ wcmatch = "8.4"
 avro = { version = "~1.11.2", optional = true }
 cohere = { version = "4.21", optional = true }
 fastavro = { version = "~1.8.0", optional = true }
-langchain = { version = "0.0.271", optional = true }
+langchain = { version = "0.1.16", optional = true }
```

Comment:
Should we `^0.1` it?

Reply:
That would be ideal, but a bit risky.. I am thinking we fix the destinations first, and then make another pass to bump Langchain to latest.  So we will end up with two CDK versions - one with pinned langchain (0.1.16) and one with latest langchain(^0.1). This would be helpful in case we need to rollback any of the destinations (to 0.1.16) at any point.